components/definitions/rewardsInternals.d.ts

```diff
@@ -15,7 +15,8 @@ declare namespace RewardsInternals {
     log: string
     fullLog: string
     externalWallet: ExternalWallet,
-    eventLogs: EventLog[]
+    eventLogs: EventLog[],
```

Comment:
I don't think commas are necessary in interfaces.

Reply:
Done